README.md

```diff
@@ -1 +1,95 @@
-# layout
\ No newline at end of file
+# Wolox Layout
+
+[![circle-ci](https://img.shields.io/circleci/project/github/Wolox/wolox-layout.svg)](https://circleci.com/gh/Wolox/wolox-layout)
+[![npm](https://img.shields.io/npm/v/wolox-layout.svg)](https://www.npmjs.com/package/wolox-layout)
+
+[![FEArmy](./assets/FEA_icon.png)](https://github.com/orgs/Wolox/teams/front-end-army/members)
+
+## Features
+
+* `margins.scss` & `paddings.scss` files contains all the margin and padding classes you need to start a basic project.
+* `layout.scss` contains the basic flex box properties.
+
+## Installation
+
+1.  intall the package with `npm` or `yarn`
+
+#### npm
+
+```bash
+npm i wolox-layout
+```
+
+#### yarn
+
+```bash
+yarn add wolox-layout
+```
+
+2- Include the wolox-bootstrap.scss before the index file of your CSS/SCSS.
+
+If you want to import the 'wolox-bootstrap' in this way:
+
+```js
+@import 'wolox-bootstrap';
+```
+
+You have to add the following lines to the webpack configuration file:
+
+```js
+{
+  loader: 'sass-loader',
+  options: {
+    includePaths: ['node_modules/wolox-layout']
+  }
+}
+```
+
+Otherwise you can import this file by doing:
+
+```js
+@import 'node_modules_path/wolox-layout/wolox-bootstrap';
+```
+
+Another way is to import the `margins.scss`, `paddings.scss` or `layout.scss` separately.
+
+## Usage
+
+* This will generate a margin top of 10px to your headline:
```

Comment:
we should allow scale override to allow a definition of 5px instead of 10 or stuff like that